examples/graphql-field-interceptors/graphql_field_interceptors.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/graphql;
+import ballerina/log;
+
+// Defines an interceptor `LogInterceptor` using a service class. It cannot have any
+// `resource`/`remote` methods except the `execute()` remote method. Other methods are allowed.
```

Comment:
I think we don't need the quotes here since it will be rendered as Ballerina code. Check the other places.
```suggestion
// Defines an interceptor mnamed LogInterceptor using a service class. It cannot have any
// resource/remote methods except the `execute()` remote method. Other methods are allowed.
```

Reply:
Quotes are used in other examples.

Reply:
I think they are there for historical reasons. (Previously we used to render these comments separately using MD formatting, which isn't the case now). Anyways, let's keep this for now and check back later.